src/main/java/com/marklogic/rowbot/QueryBot.java

```diff
@@ -45,8 +47,13 @@ public QueryBot(String connectionString, String user, String password, JSONObjec
 		}
 		
 		try {
-			DriverManager.registerDriver(new oracle.jdbc.driver.OracleDriver());
-			this.connection = DriverManager.getConnection(connectionString, user, password);
+			//DriverManager.registerDriver(new oracle.jdbc.driver.OracleDriver());
+			DriverManagerDataSource ds = new DriverManagerDataSource();
+			ds.setDriverClassName("oracle.jdbc.driver.OracleDriver");
+			ds.setUrl("JDBC_URL");
+			ds.setUsername("JDBC_USERNAME");
+			ds.setPassword("JDBC_PASSWORD");
+			this.connection = ds.getConnection();;
 		} catch (SQLException e) {
```

Comment:
This is the Spring stuff I mentioned.  Pass in the JDBC params as env variables.  These are hardcoded in for now.